nebula_ros/src/continental/continental_ars548_hw_interface_ros_wrapper.cpp

```diff
@@ -0,0 +1,329 @@
+    descriptor.read_only = true;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<std::string>("host_ip", "255.255.255.255", descriptor);
+    sensor_configuration.host_ip = this->get_parameter("host_ip").as_string();
+  }
+  {
+    rcl_interfaces::msg::ParameterDescriptor descriptor;
+    descriptor.type = rcl_interfaces::msg::ParameterType::PARAMETER_STRING;
+    descriptor.read_only = true;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<std::string>("sensor_ip", "192.168.1.201", descriptor);
+    sensor_configuration.sensor_ip = this->get_parameter("sensor_ip").as_string();
+  }
+  {
+    rcl_interfaces::msg::ParameterDescriptor descriptor;
+    descriptor.type = rcl_interfaces::msg::ParameterType::PARAMETER_STRING;
+    descriptor.read_only = true;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<std::string>("multicast_ip", "224.0.0.1", descriptor);
+    sensor_configuration.multicast_ip = this->get_parameter("multicast_ip").as_string();
+  }
+  {
+    rcl_interfaces::msg::ParameterDescriptor descriptor;
+    descriptor.type = rcl_interfaces::msg::ParameterType::PARAMETER_STRING;
+    descriptor.read_only = false;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<std::string>("frame_id", "continental", descriptor);
+    sensor_configuration.frame_id = this->get_parameter("frame_id").as_string();
+  }
+  {
+    rcl_interfaces::msg::ParameterDescriptor descriptor;
+    descriptor.type = rcl_interfaces::msg::ParameterType::PARAMETER_INTEGER;
+    descriptor.read_only = true;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<uint16_t>("data_port", 2368, descriptor);
+    sensor_configuration.data_port = this->get_parameter("data_port").as_int();
+  }
+  {
+    rcl_interfaces::msg::ParameterDescriptor descriptor;
+    descriptor.type = rcl_interfaces::msg::ParameterType::PARAMETER_INTEGER;
+    descriptor.read_only = true;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<uint16_t>("configuration_host_port", 2368, descriptor);
+    sensor_configuration.configuration_host_port =
+      this->get_parameter("configuration_host_port").as_int();
+  }
+  {
+    rcl_interfaces::msg::ParameterDescriptor descriptor;
+    descriptor.type = rcl_interfaces::msg::ParameterType::PARAMETER_INTEGER;
+    descriptor.read_only = true;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<uint16_t>("configuration_sensor_port", 2368, descriptor);
+    sensor_configuration.configuration_sensor_port =
+      this->get_parameter("configuration_sensor_port").as_int();
+  }
```

Comment:
Same as for the `decoder_ros_wrapper`, please adapt the defaults to the ones in the launch file, or, even better, remove them.

Reply:
Addressed in [31703e5](https://github.com/tier4/nebula/pull/107/commits/31703e5ab2c252069ac509627a2d002f7ba1dd35)

Just in case I will try with the device tomorrow